README.md

```diff
@@ -0,0 +1,19 @@
+# Zirgen Circuit Compiler
+
+Zirgen is a compiler for a domain-specific language, also called "zirgen",
+which creates arithmetic circuits for the RISC Zero proof system.
+
+For the most part, users of RISC Zero should be writing their zkApps in Rust
+and using our RISC-V zkVM rather than building them directly in Zirgen.
+Sometimes, however, it is necessary or desirable to write parts of an
+application as an arithmetic circuit to integrate directly with our proof system
+and achieve better performance. In fact, our zkVM itself is a circuit written
+in Zirgen! Zirgen makes it possible to build accelerators for important parts
+of your applications, or even to build entire other VMs that integrate into the
+RISC Zero ecosystem through proof composition. If that sounds like your goal,
```

Comment:
hmm... i'd like to be careful to avoid indicating that this repo is ready for external users to build accelerator circuits for the zkVM. can we add verbiage to indicate the level of maturity and usability of the contents of this repo?

Reply:
also i think it would be worthwhile to include information about the plan to support circom integration -- IIUC users are going to be able to integrate circom-built accelerator circuits before they can integrate zirgen-built acccelerator circuits.